bootstrap/scripts/2-download.sh

```diff
@@ -14,8 +14,12 @@ SCRIPTS="$(cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P)"
 mkdir -p "${BOOTSTRAP_CACHE}" #required to generate hermes files
 
 ${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 70
-wget --progress=dot:mega https://github.com/carolahp/PharoBootstrap/releases/download/v1.7.0/bootstrapImage.zip
-unzip bootstrapImage.zip
+wget --progress=dot:mega https://github.com/hogoww/BootstrapImage/archive/master.zip
```

Comment:
Please, stop having different versions of these things everywhere :(
The bootstrap image should not change repository every 2 commits.

`https://github.com/hogoww/BootstrapImage`

Reply:
Agreed. I did not know how put it as a ressource on Github, so I did what I had to do to make it work.
I did that over christmas break, alone, with zero knowledge of the boostrap. 
Point was to get it done, and see that it worked.